sphinx/source/docs/user_guide/source_examples/interaction_callbacks_in_python.py

```diff
@@ -0,0 +1,28 @@
+from bokeh.io import vform
+from bokeh.models import CustomJS, ColumnDataSource, Slider
+from bokeh.plotting import figure, output_file, show
+
+output_file("callback.html")
+
+x = [x*0.005 for x in range(0, 200)]
+y = x
+
+source = ColumnDataSource(data=dict(x=x, y=y))
+
+plot = figure(plot_width=400, plot_height=400)
+plot.line('x', 'y', source=source, line_width=3, line_alpha=0.6)
+
+@CustomJS
```

Comment:
Hmm, I understand this but I'm not sure I like this exactly as-is. It's a little surprising to see a class used as a decorator, and also CamelCase for decorators is jarring as well. It's also not "wrapping" the function and returning a new function, which is the typical usage people would expect. I would say show explicit usage by created the a `CustomJS` and passing it `callback` as an argument.